config/config.exs

```diff
@@ -2,6 +2,8 @@
 # and its dependencies with the aid of the Mix.Config module.
 use Mix.Config
 
+config :nerves_runtime, :disable_syslog, Mix.env() == :test
```

Comment:
Do people think this is a good idea? I mainly wanted to just have a way to not try to start the `GenServer`s up when you're running in CI, but it's also an accidental bonus feature that people could disable this logging if they didn't want it for some reason.

Reply:
This seemed like a good idea to me.

Reply:
Agreed.  If it were me, I would probably name it in the enabled sense (enable_syslog) and use pattern matching in function heads instead of checking with `if`.  (On phone or else I would give an example.)